### PR TITLE
Put BaseUrl in config

### PR DESCRIPTION
**Pull request by pallenSL**

### CODE REVIEW

File: Auth.Web/App.config [hunk 1]
```diff
@@ -7,5 +7,6 @@
     <add key="AWSProfileName" value="dynamo-db-write"/>
     <add key="serilog:minimum-level" value="Debug" />
     <add key="serilog:write-to:ColoredConsole" />
+    <add key="BaseUrl" value="http://*:9003"/>
```
Comment (csscottlogic) [inline Auth.Web/App.config]: Should this be listening on 8080 now that it's a container in an overlay network?

Comment (pallenSL) [inline Auth.Web/App.config]: It was 9003 in the code.
That is possibly wrong as it isn't currently used.
Can change.

Comment (csscottlogic) [inline Auth.Web/App.config]: OK

**Comment by csscottlogic** [comment]
:shipit:, after port change and solving merge conflicts

**Pull request merged**